### PR TITLE
web font loading enhancement 

### PR DESCRIPTION
**Pull request by barry800414**
move font loading css to the tail of body, not in header, because loading css will block browser

reference: 
https://www.keycdn.com/blog/web-font-performance/

### CODE REVIEW

**Comment by barry800414** [comment]
#37

File: css/grayscale.css [hunk 1]
```diff
@@ -4,12 +4,11 @@
  * For details, see http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-@import url(https://fonts.googleapis.com/earlyaccess/notosanstc.css);
 
 body {
     width: 100%;
     height: 100%;
-    font-family: 'Noto Sans TC', "Helvetica Neue",Helvetica,Arial,sans-serif;
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
```
Comment (mark86092) [inline css/grayscale.css:11]: 其實可以移除這個沒差，bootstrap 有定最基本字型

File: css/font.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+
```
Comment (mark86092) [inline css/font.css]: 上面兩行刪掉

**Comment by mark86092** [comment]
@barry800414: dos2unix and no x file permission, thanks

File: css/grayscale.css (hunk 1 above)
Comment (barry800414) [inline css/grayscale.css:11]: 沒差啊~~  還沒讀到就用這個template的預設字型?

File: css/font.css (hunk 2 above)
Comment (barry800414) [inline css/font.css]: ok

**Pull request merged**